src/Mvc/Mvc.NewtonsoftJson/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson.csproj

```diff
@@ -21,7 +21,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\Mvc.Core\src\Formatters\ResponseContentTypeHelper.cs" />
```

Comment:
Is there a _new_ `InternalsVisibleTo` relationship that makes this source file unnecessary?

Reply:
What is the answer to my question / resolution here?

Reply:
Oh, I see. File no longer changed.